parquet/src/main/java/org/apache/iceberg/parquet/ParquetBloomRowGroupFilter.java

```diff
@@ -290,7 +299,7 @@ private <T> boolean shouldRead(
               hashValue = bloom.hash(((Number) value).intValue());
               return bloom.findHash(hashValue);
             default:
-              return ROWS_MIGHT_MATCH;
+              return true;
```

Comment:
I think it is more readable to keep the constants:
```suggestion
              return ROWS_MIGHT_MATCH;
```

A few more occurrences below

Reply:
This is because the types of these two constants have been changed from `boolean` to the generic class `Expression` of [`FindsResidualVisitor`](https://github.com/apache/iceberg/pull/10090/files#diff-0fcc434085226037ca86087d4ed36984b30def152e4649b2adcc352f16d15f31).

For readability, maybe we can add a comment after the boolean value ?
```java
return true; /* rows might match */